chain/rosetta-rpc/src/models.rs

```diff
@@ -982,27 +982,26 @@ pub(crate) struct TransactionIdentifier {
 impl TransactionIdentifier {
     /// Returns an identifier for a NEAR transaction with given hash.
     pub(crate) fn transaction(tx_hash: &near_primitives::hash::CryptoHash) -> Self {
-        Self::from_prefix_and_hash("tx", tx_hash.to_base())
+        Self::from_prefix_and_hash("tx", tx_hash)
```

Comment:
“to_base” is explicit, and given there are just a few places, I would stick to it rather than implicitly depending on Display implementation of CryptoHash

Reply:
Should we use to_base in `block_event` as well then?